sphinx_gallery/gen_gallery.py

```diff
@@ -39,6 +40,11 @@
     'abort_on_example_error': False,
     'failing_examples': {},
     'expected_failing_examples': set(),
+    'binder_url': None,
+    'binder_org': None,
+    'binder_repo': None,
+    'binder_branch': 'gh-pages',
+    'binder_reqs': '../requirements.txt',
```

Comment:
I prefer a config dictionary and a default false flag. That is, here in the defaults just a single entry:
```python
'binder': False,
```
The user will then config
```python
'binder':{'url': 'http...', 'org': '...', 'branch': '...', ...}
```